Header header

int64 sensor_id
float64 distance

int64 FRONT_LEFT  = 0
int64 FRONT_RIGHT = 1
int64 LEFT        = 2
int64 RIGHT       = 3
int64 BACK_LEFT   = 4
int64 BACK_RIGHT  = 5
